drivers/volume/portworx/schedops/k8s-schedops.go

```diff
@@ -335,6 +335,10 @@ func isMountReadOnly(mount string) bool {
 	return re.MatchString(mount)
 }
 
+func isErrPodNotFound(err error) bool {
```

Comment:
The `container not found` check is not the right thing to do based on the method name. `isErrPodNotFound` should return only if the pod is actually not present. 

`container not found` means that a particular container in a pod is not found. The pod itself is present.

Reply:
yeah container not found happens when a pod is in a not ready state

Reply:
Exactly. `Completed` is not the same as `isErrPodNotFound`. The helper function name can mislead future users of the function.